static/css/style.css

```diff
@@ -45,6 +45,7 @@
   --red-complementary-color: #f72823;
   --blue-complementary-color: #4be0fe;
   --green-complementary-color: #23f751;
+  --white: #FFFFFF;
```

Comment:
nit: We used lowercase for all other color, I'd follow the same convention for this as well.